### PR TITLE
Folder Restructuring 2

### PR DESCRIPTION
**Pull request by RandomAshes**
## Issue
Resolves N/A

## What was done
- [x] Restructured the Assignment, ModalFormComponent, and Auth component folder structures

## How to test
1. Nothing functionally should have changed.

### CODE REVIEW

File: ui/src/Styles/TagRowsContainer.scss [hunk 1]
```diff
@@ -15,8 +15,8 @@
  * limitations under the License.
  */
 
-@import '../Styles/Main.scss';
-@import '../Styles/Breakpoints.scss';
+@import 'Main';
+@import 'Breakpoints';
```
Comment (hirava271) [inline ui/src/Styles/TagRowsContainer.scss:19]: Interesting! I assume compiler looks for the file name in entire project to find the correct one to import?

**Review by hirava271 — APPROVED** [review]
This reorganization of files makes sense to me! There are still some folders left so I assume there will be folder-restructure-part-3 coming?

**Review by griffico — COMMENTED** [review]
I find the move of the Assignment Client and the Service into the API folder a little strange, given that they only have to do with Assignments. I like the Common folder though

**Comment by griffico** [comment]
> I find the move of the Assignment Client and the Service into the API folder a little strange, given that they only have to do with Assignments. I like the Common folder though

Based on Hirava's comment, I'm assuming there are more of these that have names I don't remember?

**Review by aclift19 — APPROVED** [review]

**Comment by RandomAshes** [comment]
@hirava271 Yes, there's another PR that's gonna go in right after this.

@egriff38ford I agree about the services being moved. I kinda did it as a temporary thing as there's not a good local spot for those kinda helper services. In my next PR I get rid of domain specific folders altogether. I didn't feel like the common folder felt right for those kinda services. I plan to come back to them at a later time, so was just throwing them in a central location. Lmk if you have thoughts about what to do with them. I could put those kinda functions in the client services, but idk.

**Comment by griffico** [comment]
@RandomAshes you should probably add more description to the intent behind this particular refactor into the description of the PR, if not the title (if that's even possible). That would make it easier to decipher which parts we should be checking in on, and which parts we can assume are still in progress

**Pull request merged**